src/main/java/wolox/training/models/Book

```diff
@@ -0,0 +1,128 @@
+package wolox.training.models;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = true)
+    private String genre;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    @Column(nullable = false)
+    private int pages;
+
+    public Book(String bookTitle, String bookAuthor, String bookGenre, String bookImage, String bookSubtitle, String bookYear, String bookPublisher, String bookIsbn, int bookPages) {
+        title = bookTitle;
```

Comment:
Use setters in the constructor to set values so that, if at some point there are validations to be made on the arguments, that logic can reside only in the setters.

Reply:
I'm not pretty sure if I understood the change you requested. Could you put an example to me or recommend me some doc to read about it?

Reply:
Instead of title = bookTitle, do setBookTitle(bookTitle)